js_modules/dagster-ui/packages/ui-core/src/graph/KindTags.tsx

```diff
@@ -62,17 +59,14 @@ export const AssetComputeKindTag = ({
         style={{...style, cursor: shouldLink || currentPageFilter ? 'pointer' : 'default'}}
         tags={[
           {
-            label: definition.computeKind,
-            onClick:
-              currentPageFilter && definition.computeKind
-                ? () => currentPageFilter.setState(new Set([definition.computeKind || '']))
-                : shouldLink
-                ? () => {
-                    window.location.href = linkToAssetTableWithComputeKindFilter(
-                      definition.computeKind || '',
-                    );
-                  }
-                : () => {},
+            label: computeKind,
+            onClick: currentPageFilter
+              ? () => currentPageFilter.setState(new Set([computeKind || '']))
```

Comment:
Hmm so this PR updates the rendering to fall back to the tag when the definition.computeKind is not present, but does the on-page / asset table filtering do the same?  I think technically you could be seeing a "Snowflake" tag, and click in to one of these filtered views, and no longer see the asset unless there are changes to the filtering to match the display changes in this PR?

(Disregard if that's been done elsewhere!)